doc/devel/coding_guide.rst

```diff
@@ -156,6 +157,12 @@ Content topics:
 * Does the PR conform with the :ref:`coding_guidelines`?
 * Is the :ref:`documentation <pr-documentation>` (docstrings, examples,
   what's new, API changes) updated?
+* Is the change only reflowing code/docstrings for increased line length?
+  Generally, such changes are discouraged when not part of other non-stylistic
+  work because it obscures the git history of functional changes to the code.
+  Reflowing a method or docstring as part of a larger refactor/rewrite is
+  acceptable.
```

Comment:
This seemed a little out of left-field - maybe more general is better?  

```suggestion
* Is the change just stylistic?  Generally, such changes are discouraged when not 
  part of other non-stylistic work because it obscures the git history of functional 
  changes to the code. Style changes to a method or docstring as part of a larger 
  refactor/rewrite is acceptable.
```
```suggestion
* Is the change only reflowing code/docstrings for increased line length?
  Generally, such changes are discouraged when not part of other non-stylistic
  work because it obscures the git history of functional changes to the code.
  Reflowing a method or docstring as part of a larger refactor/rewrite is
  acceptable.
```